src/client/assets/main.coffee

```diff
@@ -31,25 +31,60 @@ class Router extends Backbone.Router
     'settings/authors': 'authorsEdit'
 
   articleEdit: ->
-    require('../apps/edit/client').init()
+    require.ensure(
```

Comment:
🎉 

Side note: if this was JS/TS we could use `import()` here

Reply:
Would like to get this router moved out of backbone, but it seems a chore in itself -- can look into tackling this and converting in a separate pr.